util-taglib/src/com/liferay/taglib/portletext/RuntimeTag.java

```diff
@@ -260,8 +260,9 @@ public static void doTag(
 					httpServletRequest,
 					"the-application-cannot-include-itself");
 
-				httpServletRequest.setAttribute(
-					"liferay-portlet:runtime:errorMessage", errorMessage);
+				pageContext.setAttribute(
```

Comment:
@dantewang @yuhai I do not understand why this is related to this issue.
Is this issue exposed by your changes in last commit?

Reply:
@tinatian, without the fix, the issue also existed. the errorMessage is null in UI. I added it because the current fix also is related with the output result(errorMessage) in UI.

Reply:
@yuhai Ping me when you are up, I would like to have a talk to know more details about this.